src/main/java/lotto/view/LottoResultView.java

```diff
@@ -0,0 +1,90 @@
+package lotto.view;
+
+import java.util.Collections;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+import lotto.domain.Lotto;
+import lotto.domain.LottoNumber;
+import lotto.domain.Lottos;
+import lotto.domain.Ranking;
+
+public class LottoResultView {
+
+    private static final int ZERO = 0;
+    private static final String LOTTO_STATISTICS_MESSAGE = "당첨 통계";
+    private static final String HYPHEN_MESSAGE = "---------";
+    private static final String TOTAL_YIELD_MESSAGE = "총 수익률은 ";
+    private static final String FINISH_MESSAGE = "입니다.";
+    private static final String OPENING_PARENTHESIS = "(";
+    private static final String MATCH_AMOUNT_MESSAGE = "개 일치";
+    private static final String BONUS_BALL_MATCH_MESSAGE = "보너스 볼 일치";
+    private static final String WON_MESSAGE = "원) - ";
+    private static final String AMOUNT_MESSAGE = "개";
+    private static final String SPACE = " ";
+    private static final String COMMA = ",";
+    private static final String LOTTO_AMOUNT_MESSAGE = "개를 구매했습니다.";
+
+    public void printLottos(final Lottos lottos) {
+        printAmount(lottos);
+        for (Lotto lotto : lottos.getLottos()) {
+            printLotto(lotto.getLottoNumbers());
+        }
+    }
+
+    private void printAmount(final Lottos lottos) {
+        System.out.println(lottos.getLottos().size() + LOTTO_AMOUNT_MESSAGE);
+    }
+
+    private void printLotto(final List<LottoNumber> lottoNumbers) {
+        final List<Integer> numbers = lottoNumbers.stream()
+            .map(LottoNumber::getNumber)
+            .collect(Collectors.toList());
+
+        System.out.println(numbers);
+    }
+
+    public void finishGame(final Map<Ranking, Integer> totalResult, final double totalWinnerPrize) {
+        System.out.println(LOTTO_STATISTICS_MESSAGE);
+        System.out.println(HYPHEN_MESSAGE);
+
+        final List<Ranking> rankings = new LinkedList<>(totalResult.keySet());
+        Collections.sort(rankings);
+        Collections.reverse(rankings);
```

Comment:
혹시라도 `내림차순`으로 변경하고 싶으신거라면 아래와 같이 변경해볼 수 있을 것 같아요! 😃 
```suggestion
        Collections.sort(rankings, Comparator.reverseOrder());
```

Reply:
내림차순 정렬을 의도한게 맞습니다!!

`Comparator`가 있었군요!! 감사합니다~!🙇🏻